### ISSUE TITLE
Default search_index is created as btree

### ISSUE THREAD

**Issue by Nepherhotep** [linuxlewis/djorm-ext-pgfulltext#61]
Default search_index created using btree:

CREATE INDEX tablename_a71a185f
  ON "tablename"
  USING btree
  (search_index);

Shouldn't it use gin or gist instead? Is it possible to configure vector field to use them, or only further migration altering it will work?

P.S. Maybe issue tracker is not the perfect place for such discussions, but it might be useful for community if I create pull request extending readme with answer.

**Comment by vladiibine** [comment]
Agreed. GIN or GIST would be exactly the index types I'd use, so this would be a nice feature to have.

**Comment by hyusetiawan** [comment]
any update on this?

**Comment by Nepherhotep** [comment]
No, currently Django doesn't have native support of functional indexes, so the only way to do it - not to use this library and write a custom migration.
However, since Django 1.9 it will support indexes other than BTREE (if we are lucky :), so maybe this library can be easily extended as well.
